SConstruct

```diff
@@ -741,7 +741,7 @@ if selected_platform in platform_list:
     if env["minizip"]:
         env.Append(CPPDEFINES=["MINIZIP_ENABLED"])
 
-    editor_module_list = ["freetype"]
+    editor_module_list = []
```

Comment:
I guess we don't really need the `editor_module_list` logic now.

That being said, we could decide to make both `regex` and `svg` modules mandatory for the editor so I guess we can keep it for later use.